pkg/infra/x/nanogit/testdata/tree-ryantxu-test-repo-export-0003.json

```diff
@@ -0,0 +1,74 @@
+{
+  "branch": "79e6a54ef3b5b51b5235374f9fef652ccb40ca44",
+  "root": {
+    "hash": "9a1094fbd0ec17e7011cebba650b659395920c8a",
+    "folder": true,
+    "children": [
+      {
+        "name": "org1",
+        "hash": "9ca9658f702517e29edfdf6f314156e3ef3c80be",
+        "folder": true,
+        "children": [
+          {
+            "name": "aaa.txt",
+            "hash": "7284ab4d2836271d66b988ae7d037bd6ef0d5d15",
+            "shared": true
+          },
+          {
+            "name": "aaa2.txt",
+            "hash": "7284ab4d2836271d66b988ae7d037bd6ef0d5d15",
+            "shared": true
+          },
+          {
+            "name": "bbb.txt",
+            "hash": "6484fb6f9cea3887578def1ba0aa96fcce279f5b",
+            "shared": true
+          },
+          {
+            "name": "sub",
+            "hash": "06ecf670d585639c2448c0fe9bcc84d9b5584547",
+            "shared": true,
+            "folder": true
```

Comment:
This caused me the most confusion -- when parts of the tree are duplicated, the "folders" (yes i know they are not _really_ folders) are just references to the same object.  This JSON structure indicates the objects that are used more than once, and keeps a single instance of the subtree saved in the "shared" section